app/helpers/teacher_helper.rb

```diff
@@ -9,6 +9,21 @@ def teacher_trn(teacher)
     "TRN: #{teacher.trn}"
   end
 
+  # @param teacher [Teacher]
+  def teacher_induction_start_date(teacher)
+    Teachers::InductionPeriod.new(teacher).formatted_induction_start_date
+  end
+
+  # @param teacher [Teacher]
+  def teacher_induction_programme(teacher)
+    Teachers::InductionPeriod.new(teacher).induction_programme
+  end
+
+  # @param teacher [Teacher]
+  def teacher_induction_ab_name(teacher)
+    Teachers::InductionPeriod.new(teacher).appropriate_body_name
+  end
```

Comment:
should we add `reported_by_ab` to these names to prevent confusion with those reported by the school?

Reply:
I think we will need something, not sure what's best at the moment; thinking it'll be easier when we have all of it populated and in use. Some naming convention, or perhaps a single point of entry where you can ask for a specific "reporter" could be useful.